sound/soc/intel/boards/Kconfig

```diff
@@ -547,7 +548,7 @@ config SND_SOC_INTEL_EHL_RT5660_MACH
 	tristate "EHL with RT5660 in I2S mode"
 	depends on I2C && ACPI
 	depends on MFD_INTEL_LPSS || COMPILE_TEST
-	depends on SND_HDA_CODEC_HDMI
+	depends on SND_HDA_CODEC_HDMI && SND_SOC_SOF_HDA_AUDIO_CODEC
```

Comment:
Same comment as for JSL change.
This not in Mark's tree either, so can you put these in a separate patch?

Reply:
OK. I will make a separate patch for it. And for the "if" statement, please see my previous comments.

Reply:
Wait a sec, I just reviewed #1996 from Pierre and Pierre has put his changes in one patch, so maybe this is needed. Then this just means we need to send the machine driver first, and only then can send this fix, but should be ok.

Reply:
@kv2019i Both are OK to me : ) I will make it in the same patch.